docs/example-narratives/examples/realtime-limit-set-detailed.json

```diff
@@ -1,7 +1,10 @@
 {
   "snapshot-header": {
-    "last-updated": "2023-07-12T16:00:00-07:00",
-    "snapshot-provider": "X-RC",
+    "source": {
+      "last-updated": "2023-07-12T15:05:43.044267100-07:00",
+      "provider": "X-AMPL",
```

Comment:
I'm really hoping to focus on improving the quality of examples, and I intend to use `X-AMPL-TO` and `X-AMPL-RC` as entity ids.